docs/src/configuration/characters.md

```diff
@@ -9,7 +9,7 @@ Configurable fields:
 - `base_size`: Size of the character for collision math
 - `base_health`: Base health of the character
 - `skills`: This is a map of integer (as string) to skills, the integers represent the id and ordering of skills for calling them
-- `max_inventory_size`: Maximum amount of loots that the character can hold in their inventory
+- `burst_loads`: Number of times the character's basic skill can be used in a row before going on cooldown
```

Comment:
This should not be a property of the character as they have no concepts of "basic skill"

This property should go in the skill

Reply:
I added the property to the skill and removed it from the character. Thanks for the suggestion, I think it's much more adequate and extensible to other skills now!